pages/contact/age/index.js

```diff
@@ -0,0 +1,21 @@
+import ContactNavigation from "../../../components/layout/ContactNavigation";
+
+import classes from "../../../styles/aboutUs.module.css";
+
+function age() {
+    return <div>
+        <ContactNavigation />
+        <div className={classes.aboutUsSpace}>
+            <div className={classes.aboutUsItem}>
+                <h1>test</h1>
+            </div>
+        </div>
+    </div>
```

Comment:
I would recommend enabling automatic formatting in your editor, which would change the formatting of this to something like:

```tsx
return (
  <div>
    <ContactNavigation/>
    <div className={classes.aboutUsSpace}>
      <div className={classes.aboutUsItem}>
        <h1>test</h1>
      </div>
    </div>
  </div>
);
```

Note how everything is wrapped in parenthesis to allow the JSX to begin on the next line (not the same line as `return`). This makes the code easier for visual readers to read, because it means the opening `<div>` and closing `</div>` appear in the same column, and will be easier for the eye to locate.